src/CommunityToolkit.Maui.UnitTests/Mocks/MockRouteFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommunityToolkit.Maui.UnitTests.Mocks;
+
+public class MockRouteFactory : RouteFactory
```

Comment:
Lets add `MockPage` to the class name to better indicate that this `RouteFactory` returns a `MockPage`

```suggestion
public class MockPageRouteFactory : RouteFactory
```

Reply:
This change will be present in my update to the PR.